### PR TITLE
Add leads to leadership page

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: layouts/shortcodes/leadership.html [hunk 1]
```diff
@@ -36,12 +40,49 @@ <h4>Ed Warnicke</h4>
...
+    <img src="/img/headshots/lkunz.png" class="rounded-circle" title="Larry Kunz" />
+      <h4>Larry Kunz</h4>
+      <p>Information Architect <br/> Extreme Networks</p>
+      <p>Language Workstream Lead</p>
+  </div>
+
+  <div class="mt-4 text-center col col-md-4 col-6">
+    <img src="/img/headshots/jlee.png" class="rounded-circle" title="Joanna Lee" />
+      <h4>Joanna Lee</h4>
+      <p>Partner <br/> Gesmer Upgrove LLP</p>
```
Comment (justaugustus) [inline layouts/shortcodes/leadership.html:70]:
```suggestion
      <p>Partner <br/> Gesmer Updegrove LLP</p>
```

**Comment by mschnoor-intel** [comment]
Please fix Jim's name spelling: should be Jim St. Leger.

File: layouts/shortcodes/leadership.html (hunk 1 above)
Comment (celestehorgan) [reply layouts/shortcodes/leadership.html]: Corrected in latest force-push of commit.

**Comment by celestehorgan** [comment]
Both of the above are fixed in the latest push

**Pull request merged**